### DISCUSSION TITLE
Modbus via PocketWifi V3

### DISCUSSION THREAD

**Discussion by eyeisystems** [wills106/homeassistant-solax-modbus#496] [General]
Hi, 

I'm new to this and doing some research, I have a X1 G4 with a PocketWifi V3 and it appears i can access modbus over it? Are you aware of this? If this is the case is there a need for a serial modbus adapter? Or is it that you can only read via the PocketWifi V3 and not write?

I managed to connect to the PocketWifi IP address with Modbus Poll and could see data.

https://kb.solaxpower.com/data/detail/ff8080818407e2a701840a22dec20032.html

Forgive me if i've missed some info on this as it's all a bit of a minefield

**Comment by wills106** [comment]
You can read and write with the PocketWiFi 3.0
But overall connectivity isn't always reliable, it depends on the Firmware version.

I don't own any of the PocketWiFi range so I don't know anything about the Firmware versions.
